HEN_HOUSE/egs++/egs_application.h

```diff
@@ -1122,7 +1122,13 @@ class EGS_EXPORT EGS_Application {
     };
     virtual EGS_Float getRM() {
         return -1.0;
-    };
+    }
```

Comment:
This line is missing a semicolon.

Reply:
Thanks, @rtowson.  There were some other issues as well.  Silly git!  See the amended commit.